src/components/site/CSSPreviewConnect.tsx

```diff
@@ -0,0 +1,47 @@
+"use client"
```

Comment:
The saved custom CSS in `src/app/site/[site]/layout.tsx` will still exist and will affect the final result.

Reply:
Ok, I move to Root Style component to dot this